### PR TITLE
Adding in Persona Bar Hover Menus + Fix for Removing Edit bar on Cancel-Edit-Button-Click

### PR DESCRIPTION
**Pull request by CBPSC**
This update provides hover menus for the persona bar

Right now we have 3 icons (Content, Manage, Settings), but hopefully the Persona Bar would be extensible meaning other developers could add items into it. 

In that case I'd need to account for the new items developers added for some of the menu show/hide actions. To do that I'd make an array of Persona Bar items and loop through them so that when a menu item is hovered over... then the user goes directly above or below with the mouse to another menu icon item... the (newly added in item) would not show it's menu. 

This would be added to the closeAllMenus() function

### CODE REVIEW

File: src/Dnn.vNext/Pages/_Admin.cshtml [hunk 1]
```diff
@@ -95,16 +95,73 @@
...
+            $('#persona-cancel-edit-button').click(function () {
+                $('#persona-edit-button').show();
+                $('#persona-cancel-edit-button').hide();
+                $('footer').remove();
+            });
+        });
+    </script>
+
+
+    <script type="text/javascript">
```
Comment (SkyeHoefling) [inline src/Dnn.vNext/Pages/_Admin.cshtml]: We should be able to put this all in one `<script>` block

File: src/Dnn.vNext/Pages/_Admin.cshtml [hunk 2]
```diff
@@ -95,16 +95,73 @@
...
+        var closeAllMenus = function(){
```
Comment (SkyeHoefling) [inline src/Dnn.vNext/Pages/_Admin.cshtml]: This is now a global variable, was that intentional? I would wrap this inside of `$(function() { });` or `$(document).ready(function() { });` so the variable isn't a global. the 2nd option here using the ready function is the preferred approach I need to go through and update the other instances throughout the code.

**Comment by SkyeHoefling** [comment]
Overall it looks good, but there are a couple things that need address:

* see review comments on code
* please merge with master since this branch is conflicting

File: src/Dnn.vNext/Pages/_Admin.cshtml (hunk 1 above)
Comment (CBPSC) [reply src/Dnn.vNext/Pages/_Admin.cshtml]: I have updated the code and put into one code block

File: src/Dnn.vNext/Pages/_Admin.cshtml (hunk 2 above)
Comment (CBPSC) [reply src/Dnn.vNext/Pages/_Admin.cshtml]: Not intentional... have updated it for my script, but you may want to move it to a higher level as you noted.

File: src/Dnn.vNext/Pages/_Admin.cshtml [hunk 3]
```diff
@@ -70,23 +70,102 @@
     </environment>
 
     @RenderSection("Scripts", required: false)
+    @Html.Partial("_AddNewModule", new NewModule
```
Comment (SkyeHoefling) [inline src/Dnn.vNext/Pages/_Admin.cshtml]: This code block needs to be removed, it is from an older version of the code

**Comment by SkyeHoefling** [comment]
lookgs good

**Pull request merged**